### PR TITLE
Qaaut6 131 add eagles tests

### PR DESCRIPTION
**Pull request by vadlvadl**

### CODE REVIEW

File: src/test/java/pages/IssuePage.java [hunk 1]
```diff
@@ -9,20 +10,24 @@
...
-    String issueActionsTextXPath = "//*[@id='issue_actions_container']//folowing::[@class='action-body flooded']";
-    String priorityElementID = "priority-val";
-    String priorityFieldID = "priority-field";
-    String priorityFormSubmitButtonXPath = "//form[@id='priority-form']/div[@class='save-options']/button[@type='submit']";
+    private String commentButtonID = "footer-comment-button";
+    private String textTabButtonID = "aui-uid-1";
+    private String commentTextAreaXPath = "//*[@id='comment-wiki-edit']/textarea";
+    private String addCommentSubmitButtonID = "issue-comment-add-submit";
+    private String issueActionsContainerID = "issue_actions_container";
+    private String issueActionsTextXPath = "//*[@id='issue_actions_container']//folowing::[@class='action-body flooded']";
```
Comment (Artem100) [inline src/test/java/pages/IssuePage.java]: Reduce your Xpath request.

File: src/test/java/pages/IssuePage.java [hunk 2]
```diff
@@ -9,20 +10,24 @@
...
+    private String priorityElementID = "priority-val";
+    private String priorityFieldID = "priority-field";
+    private String priorityFormSubmitButtonXPath = "//form[@id='priority-form']/div[@class='save-options']/button[@type='submit']";
```
Comment (Artem100) [inline src/test/java/pages/IssuePage.java]: Reduce your Xpath request

File: src/test/java/pages/IssuePage.java [hunk 3]
```diff
@@ -9,20 +10,24 @@
...
 
     public void navigateTo(String url){
         open(url);
     }
```
Comment (Artem100) [inline src/test/java/pages/IssuePage.java]: Rewrite your methods like:
    public void navigateTo(String url){ open(url); }

File: src/test/java/pages/LoginPage.java [hunk 4]
```diff
@@ -18,8 +20,11 @@ public void enterPassword(String password){
...
 
-    public boolean atRequiredPage() //проверяет, что мы на dashboardPage.
-    { Assert.assertEquals(title(), "System Dashboard - Hillel IT School JIRA");
-        return true;
+    public boolean atRequiredPage(){
+        return url().equalsIgnoreCase(loginPageURL);
+    }
+
+    public void navigate(){
+        open(loginPageURL);
```
Comment (Artem100) [inline src/test/java/pages/LoginPage.java]: Rewrite your methods like:
public void navigateTo(String url){ open(url); }

File: testng.xml [hunk 5]
```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suite name="BLR_Analytics" verbose="10" parallel="tests">
 
-	<test name="Search in Jira" group-by-instances="true" verbose="3">
+	<test name="NewIssue" group-by-instances="true" verbose="3">
     		<classes>
-				<class name="tests.SearchJira"/>
```
Comment (Artem100) [inline testng.xml]: Return please class with test for Search page

Comment (vadlvadl) [reply testng.xml]: What class do you mean?  In master branch I see only 
<test name="Search in Jira" group-by-instances="true" verbose="3">
    		<classes>
				<class name="tests.SearchJira"/>
		    </classes>
</test>
These lines are present in testng.xml. They were moved from 6 to 10 line

**Pull request merged**